src/components/molecules/CookieModal/index.jsx

```diff
@@ -38,18 +69,63 @@ class ModalPage extends React.Component {
       <MDBModal
         isOpen={this.state.modal1}
         toggle={this.toggle(1)}
-        frame
-        position="bottom"
+        disableFocusTrap={false}
+        keyboard={false}
+        className="cookie-modal"
       >
         <MDBModalBody className="text-center py-4">
+          <h2>Cookie Einstellungen</h2>
+          <p className="text-muted">
+            Wir verwenden auf unserer Website Cookies, um die Benutzererfahrung
+            zu verbessern. Einige davon sind essenziell für den Betrieb der
+            Website.
+          </p>
+          <div className="my-4">
+            <MDBInput
+              label="Essenziell"
+              filled
+              checked={this.state.essential}
+              disabled
+              type="checkbox"
+              id="cookieEssenziell"
+              containerClass="mr-5"
+            />
+            <MDBFormInline className="d-flex justify-content-center">
+              <MDBInput
+                label="Persönliche Ansprache"
+                filled
+                checked={this.state.marketing}
+                name="marketing"
+                color="primary"
+                type="checkbox"
+                id="cookieMarketing"
+                containerClass="mr-5"
+                onChange={(e) => this.checkBoxCheck(e)}
+              />
+              <MDBInput
+                label="Statistiken"
+                filled
+                checked={this.state.statistics}
+                name="statistics"
+                type="checkbox"
+                id="cookieStatistiken"
+                containerClass="mr-5"
+                onChange={(e) => this.checkBoxCheck(e)}
+              />
+            </MDBFormInline>
+          </div>
           <MDBRow className="justify-content-center align-items-center">
-            <p className="pt-3 pr-2 text-left">
-              Wir verwenden Cookies, um die Funktionsfähigkeit der Website
-              aufrecht zu erhalten.
-            </p>
-            <MDBBtn color="agency-red" onClick={this.toggle(1)} size="lg">
-              Ok, danke
-            </MDBBtn>
+            <MDBCol md="6">
+              <MDBBtn color="success" onClick={() => this.checkAll()}>
+                <MDBIcon icon="check-circle" />
+                Alle akzeptieren
```

Comment:
I would recommend writing this phrase in the title case.
```suggestion
                Alle Akzeptieren
```

Reply:
It does not matter, once the button text is all uppercase. I could even write `aLlE aKzEpTiErEn`

Reply:
yOu ShOuLd StILl ChOoSe A cOnSiStEnT sHaPe.